kcidb/__init__.py

```diff
@@ -5,7 +5,7 @@
 import logging
 from kcidb.misc import LIGHT_ASSERTS
 # Silence flake8 "imported but unused" warning
-from kcidb import io, db, mq, orm, oo, monitor, tests, unittest, misc # noqa
+from kcidb import io, db, mq, orm, oo, monitor, tests, unittest, misc, argparse # noqa
```

Comment:
should i remove the` # noqa` now?

Reply:
We need the `# noqa` for flake8 to ignore that we don't necessarily use the imported modules - they're needed to make life easier for the package users (what #404 is talking about). So, please don't remove it (that would likely make CI fail anyway). We could use a more specific waiver, though: `# noqa: F401`. That would be nice to do, but it would have to be done in a separate commit (maybe updating all similar `# noqa` comments).